kafka-ballerina/tests/consumer_client_tests.bal

```diff
@@ -32,8 +32,11 @@ const string SASL_USER = "admin";
 const string SASL_PASSWORD = "password";
 const string SASL_INCORRECT_PASSWORD = "incorrect_password";
 
-const string SSL_KEYSTORE_PATH = "tests/secrets/kafka.client.keystore.jks";
-const string SSL_TRUSTSTORE_PATH = "tests/secrets/kafka.client.truststore.jks";
+const string SSL_KEYSTORE_PATH = "tests/secrets/trustoresandkeystores/kafka.client.keystore.jks";
+const string SSL_TRUSTSTORE_PATH = "tests/secrets/trustoresandkeystores/kafka.client.truststore.jks";
+const string SSL_KEY_FILE_PATH = "tests/secrets/certsandkeys/client.pem";
+const string SSL_CERT_FILE_PATH = "tests/secrets/certsandkeys/client.crt";
+const string SSL_TRUSTORE_CERT_FILE_PATH = "tests/secrets/certsandkeys/trustore.crt";
```

Comment:
Shall we rename the files and update the constants as follows?
```suggestion
const string SSL_CLIENT_PRIVATE_KEY_FILE_PATH = "tests/secrets/certsandkeys/client.private.key";
const string SSL_CLIENT_PUBLIC_CERT_FILE_PATH = "tests/secrets/certsandkeys/client.public.crt";
const string SSL_BROKER_PUBLIC_CERT_FILE_PATH = "tests/secrets/certsandkeys/broker.public.crt";
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-kafka/pull/240/commits/03cbc376e1ab1d8b4f7379683ede8487c2cb0f6a